docs/core/testing/unit-testing-mstest-configure.md

```diff
@@ -100,3 +100,99 @@ Each element of the file is optional because it has a default value.
 
 </RunSettings>
 ```
+
+## testconfig.json
+
+When running your tests with MSTest, you can use a `testconfig.json` file to configure the behavior of the test runner. The `testconfig.json` file is a JSON file that contains the configuration settings for the test runner. The file is used to configure the test runner and the test execution environment. For more information, refer to [Microsoft.Testing.Platform testconfig.json documentation](unit-testing-platform-config.md#testconfigjson).
```

Comment:
My understanding is that the file name should be `<AssemblyName>.testconfig.json`. However, we have an MSBuild target in Microsoft.Testing.Platform.MSBuild that copies testconfig.json to `<AssemblyName>.testconfig.json`, right?

Do we want to document that? Are there valid supported scenarios where Microsoft.Testing.Platform.MSBuild won't be referenced and the user will have to precisely name the file as `<AssemblyName>.testconfig.json`?